Documentation/api_v3.md

```diff
@@ -1,6 +1,33 @@
 # etcd3 API
 
-TODO: API doc
+TODO: finish API doc
+
+## Key-Value API
+
+Key-Value API is used to manipulate key-value pairs stored inside etcd. The key-value API is defined as a gRPC service[kv-service]. The Key-Value pair is defined as structured data in [protobuf format][kv-proto].
```

Comment:
The link to kv-service is still malformed. Please update to:

`The key-value API is [defined as a gRPC service][kv-service].`